src/flint/fmpq.jl

```diff
@@ -169,6 +169,21 @@ Base.ceil(a::QQFieldElem) = ceil(QQFieldElem, a)
 Base.ceil(::Type{QQFieldElem}, a::QQFieldElem) = QQFieldElem(ceil(ZZRingElem, a), 1)
 Base.ceil(::Type{ZZRingElem}, a::QQFieldElem) = cdiv(numerator(a), denominator(a))
```

Comment:
Presumably you also want this?
```suggestion
#Base.ceil(::Type{ZZRingElem}, a::QQFieldElem) = cdiv(numerator(a), denominator(a))
```